src/components/ec-toaster/ec-toaster.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div
+    v-if="messages.length > 0"
+    class="ec-toaster"
+  >
+    <transition-group
+      ref="list"
+      name="list"
```

Comment:
I'd rename the transition to something like `ec-toaster-items-transition`.

Reply:
If I change it to that, the animation brakes, I cannot find why do you know how it is possible?

Reply:
Have you updated the classes at the bottom of the <style> block too?
```
.list-enter,
.list-leave-to {
  opacity: 0;
  transform: translateX(100%);
}
.list-leave-active {
  position: absolute;
}
```